javascript-source/core/misc.js

```diff
@@ -155,7 +155,7 @@
      * @param {boolean} run
      */
     function sayWithTimeout(message, run) {
-        if (((timeout + 10000) > systemTime()) || !run) {
+        if (((timeout + 10000) > $.systemTime()) || !run) {
```

Comment:
`$.` isn't needed since this function is in the local script, no point in calling the global scope for it.